transport/tchannel/header.go

```diff
@@ -43,8 +43,9 @@ const (
 	ServiceHeaderKey = "$rpc$-service"
 	// ApplicationErrorNameHeaderKey is the response header key for the application error name.
 	ApplicationErrorNameHeaderKey = "$rpc$-application-error-name"
-	// ApplicationErrorMessageHeaderKey is the response header key for the application error messages.
-	ApplicationErrorMessageHeaderKey = "$rpc$-application-error-message"
+	// ApplicationErrorDetailsHeaderKey is the response header key for the
+	// application error details string.
+	ApplicationErrorDetailsHeaderKey = "$rpc$-application-error-details"
```

Comment:
Same as above, what is the implication of this change? this is a public method, are you sure that nobody is using this?

Reply:
same as above, not a breaking change, nice catch ;) updated the description